.ci/monorepo_force_split_branch.sh

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env bash
+
+# Split specified branches of all packages of the monorepo in parallel and force push it to corresponding repositories.
+#
+# Logs of history rewriting can be found in $WORKSPACE/split/ along with the individual bare repositories.
+#
+# Usage: monorepo_force_split_branch.sh <branch>
+#
+# Example: monorepo_force_split_branch.sh my-feature-branch
+
+# ANSI color codes
+RED="\e[31m"
+GREEN="\e[32m"
+BLUE="\e[34m"
+NC="\e[0m"
+
+SPLIT_BRANCH=$1
+WORKSPACE=${WORKSPACE:-.}
```

Comment:
```suggestion
WORKSPACE=${WORKSPACE:-$PWD}
```
if default value for $workspace is relative path and we will execute 
```sh
cd $WORKSPACE
git clone --bare .git $WORKSPACE/split/$PACKAGE
cd $WORKSPACE/split/$PACKAGE
```
in the next iteration we will not be in the original $WORKSPACE but in $WORKSPACE/split/$PACKAGE

Reply:
`monorepo_force_split_branch.sh` and `monorepo_split.sh` differ on just a few places.

Reply:
Thanks for noticing, I've fixed that in 11a620a